.ci/wercker.sh

```diff
@@ -61,10 +61,10 @@ no-error-pgjdbc)
   CS_POM_VERSION=$(mvn -e -q -Dexec.executable='echo' -Dexec.args='${project.version}' \
                      --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec)
   echo CS_version: ${CS_POM_VERSION}
-  checkout_from https://github.com/pgjdbc/pgjdbc.git
+  checkout_from https://github.com/Abhishek-kumar09/pgjdbc.git
   cd .ci-temp/pgjdbc
   # pgjdbc easily damage build, we should use stable versions
-  git checkout REL42.2.16
+  git checkout master
```

Comment:
REL 42.2.16 is not a branch.
Moreover, I had sent PR to master branch, did I made a mistake :(

Reply:
No mistake, release is set in stone.
Usage of master is good if sources are build able, but their CI is red.

Reply:
Already done.

Reply:
You can checkout ci changes here: https://github.com/checkstyle/checkstyle/pull/8083/commits/33189c8557120cb254f64d8d4360c98621dffbf8